fixtures/attribute-behavior/src/App.js

```diff
@@ -788,12 +785,12 @@ class App extends React.Component {
 
       // Creating globals for every single test is too slow.
       // However caching them between runs won't work for the same attribute names
-      // because warnings will be deduplicated. As a result, we only share globals
+      // because warnings will be de-duplicated. As a result, we only share globals
       // between different attribute names.
-      for (let i = 0; i < pool.length; i++) {
-        if (!pool[i].testedAttributes.has(attribute.name)) {
-          pool[i].testedAttributes.add(attribute.name);
-          return pool[i].globals;
+      for (const poolItem of pool) {
+        if (poolItem.testedAttributes.has(attribute.name)) {
```

Comment:
This is a good example of what I mean by 'subtle bugs introduced' - I think while refactoring here you dropped the `!`, which inverts the condition.